### DISCUSSION TITLE
Offline Maps available?

### DISCUSSION THREAD

**Discussion by micclass62** [devemux86/cruiser#275] [Q&A] [answered]
Hello,
as being one of the "sudden death of kurviger"-victims, I am now trying cruiser. Things look interesting (and somewhat  "well known"). 
Currently I do miss the offline maps that were available in Kurviger 1 (which fortunately still works for myself) and Kurviger 2. So are offline maps (or will be in the future) available in Cruiser?

Cheers,
Michael

**Comment by devemux86** [comment, marked as answer]
Hi @micclass62,

Offline maps are available in Cruiser with the built-in download manager.

Use the menu "Downloads | Maps" and download the regions you want.
Then you will have extra **offline** search!

<img src="https://github.com/devemux86/cruiser/assets/3484020/eff7386c-2efa-4fc0-8603-f527769addcc" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/f473fc99-64a2-469a-bdff-b5053f571797" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/414aee08-055d-438c-a21c-5452ced64906" width="33%"/>

**Reply by micclass62** [reply]
Thank you!  Found it. Great!
